src/Components/SmartComponents/Reports/Common/styles.js

```diff
@@ -20,7 +15,7 @@ export default StyleSheet.create({
         marginBottom: '24px'
     },
     bold: {
-        fontWeight: global_FontWeight_bold.value
+        fontWeight: 'var(--pf-global--FontWeight--bold)'
```

Comment:
Good idea. I think with this change we can also remove `@patternfly/react-token` can't we?

Reply:
Yes, `@patternfly/react-token` got removed in this PR too.